configuration.md.vtl

```diff
@@ -342,6 +343,26 @@ anObject.map = $objectKey1:$objectValue1, $objectKey2:$objectValue2
 ...
 ```
 
+<a name="Configuration-VariableInterpolation"></a>
+#[[######Variable Interpolation]]#
+
+You can use variable interpolation when defining values. Supported types are environment variables, system properties and constants.
+
+For constants, use `${const:CONSTANT_NAME}`, for environment variables and system properties, use `${ENV_VARIABLE_NAME}` or ${`PROPERTY`}.
```

Comment:
```suggestion
For constants, use `${const:com.example.YourClass.CONSTANT_NAME}`, for environment variables and system properties, use `${ENV_VARIABLE_NAME}` or `${system.property}`.
```

Use fully qualified name for const example, and lower case for system properties, as that is _normally_ the convention